admin/commands/summary.go

```diff
@@ -85,7 +85,11 @@ func addFile(zipW *zip.Writer, name string, fileName string) {
 		logrus.Debugf("%s", err)
 		r = io.NopCloser(bytes.NewReader([]byte(err.Error() + "\n")))
 	}
-	defer r.Close() //nolint:gosec,errcheck
+	defer func() {
+		if e := r.Close(); e != nil {
+			err = e
+		}
+	}()
```

Comment:
I think it's better to use nolint for such kind of cases or log the error otherwise it makes no sense

Reply:
Once I used nolint, but it failed for another rule which said `nolintlint`.
I could not avoid with `nolintlint`

Reply:
Fixed adding `nolintlint`
https://github.com/percona/pmm/pull/2545/commits/8dcf35c372017ebfd1542faac01f0b3454870b31